sdk/core/azure-core/src/main/java/com/azure/core/http/HttpHeaders.java

```diff
@@ -54,8 +54,8 @@ public HttpHeaders(Iterable<HttpHeader> headers) {
         this.headers = new HashMap<>((int) (headers.headers.size() / 0.75f));
         // TODO (alzimmer): This, and the API above, should be copying using the value list as this will String
         //  join multiple values. Or, better yet provide better copy methods on HttpHeader and Header.
-        headers.headers.forEach((key, value) ->
-            this.headers.put(key, new HttpHeader(value.getName(), value.getValue())));
+        headers.headers
+            .forEach((key, value) -> this.headers.put(key, new HttpHeader(value.getName(), value.getValue())));
```

Comment:
One thing I like to do, although IntelliJ often fights me, is to vertically align the dots.

Reply:
This is doable